x-pack/plugins/security_solution/public/resolver/store/middleware/related_events_fetcher.ts

```diff
@@ -49,6 +49,7 @@ export function RelatedEventsFetcher(
               eventCategory: newParams.panelParameters.eventCategory,
               cursor: result.nextEvent,
               nodeID,
+              lastCursorRequested: null,
```

Comment:
Can you change the name of this field to `cursorRequested` or `cursorUsedInRequest`? The reducer determines which `cursor` was used last, not the action.